espnet/bin/asr_train.py

```diff
@@ -159,6 +159,8 @@ def main(cmd_args):
                         help='Number of processes of iterator')
     parser.add_argument('--preprocess-conf', type=str, default=None,
                         help='The configuration file for the pre-processing')
+    parser.add_argument('--use-specaug', type=strtobool, default=True,
```

Comment:
I think default should be `False` at this time (please fix this in next PR @kan-bayashi @bobchennan  )